### PR TITLE
chore: Fix Structure and Bump minimum version

### PR DESCRIPTION
**Pull request by Iguins**
GitHub Issue: #

## Checklist

Please check if your PR fulfills the following requirements:

- [ ] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [x] Contains **NO** breaking changes
- [ ] Updated the Changelog
- [ ] Associated with an issue

### CODE REVIEW

File: src/MessageDialog.Uno/MessageDialog.Uno.csproj [hunk 1]
```diff
@@ -49,5 +50,5 @@
 		</None>
 	</ItemGroup>
 
-	<Import Project="..\..\Shared\Shared.projitems" Label="Shared" />
+	<Import Project="..\Shared\Shared.projitems" Label="Shared" />
```
Comment (jeanplevesque) [inline src/MessageDialog.Uno/MessageDialog.Uno.csproj]: Shouldn't it be `MessageDialog.Shared` instead of `Shared`?

Comment (jeanplevesque) [reply src/MessageDialog.Uno/MessageDialog.Uno.csproj]: For both the folder name and the label for the reference

Comment (Iguins) [reply src/MessageDialog.Uno/MessageDialog.Uno.csproj]: yep!

Comment (Iguins) [reply src/MessageDialog.Uno/MessageDialog.Uno.csproj]: Fixed!

**Review by GabrielChouinardLetourneau — APPROVED** [review]

File: src/MessageDialog.Uno/MessageDialog.Uno.csproj (hunk 1 above)
Comment (jeanplevesque) [reply src/MessageDialog.Uno/MessageDialog.Uno.csproj]: You forgot the label

File: src/MessageDialog.Shared/MessageDialog.Shared.projitems [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <SharedGUID>3ab437c2-4f5b-4e7c-ad5c-563e03fb9a4a</SharedGUID>
   </PropertyGroup>
   <PropertyGroup Label="Configuration">
-    <Import_RootNamespace>Shared</Import_RootNamespace>
+    <Import_RootNamespace>MessageDialog.Shared</Import_RootNamespace>
```
Comment (jeanplevesque) [inline src/MessageDialog.Shared/MessageDialog.Shared.projitems]: Actually, the root namespace should be `MessageDialog`, not `MessageDialog.Shared`.

Comment (Iguins) [reply src/MessageDialog.Shared/MessageDialog.Shared.projitems]: true!

**Review by jeanplevesque — APPROVED** [review]

**Pull request merged**